dev-util/lockrun/lockrun-20240415.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Lockrun - runs cronjobs with overrun protection"
+HOMEPAGE="http://www.unixwiz.net/tools/lockrun.html"
+
+SRC_URI="https://downloads.uls.co.za/gentoo/lockrun/lockrun-${PV}.c.gz"
+S="${WORKDIR}"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~x86"
+
+src_prepare() {
+	default
+	mv "${P}.c" "${PN}.c" || die
+}
+
+src_compile() {
+	emake CC="$(tc-getCC)" ${PN}
```

Comment:
Surely CC= here should not be required?

Reply:
I doubt it. It's using the built-in GNU Make recipe for a C program. Exporting `CC` worked for me on the CLI outside of portage.

Reply:
I interpret that as "rather leave it"?

Reply:
For some reason that I don't know, CC is not exported by default into the build environment (I always wondered why that is), while CFLAGS/LDFLAGS is. As a result, I think, you want to invoke `make` (or `emake` in this case), with CC set when using make to compile C code, so that the correct C-compiler executable is used.

Reply:
ACK.  Thanks for that.  Marking as resolved then.